### DISCUSSION TITLE
Famicom P2 Microphone

### DISCUSSION THREAD

**Discussion by bluestone19** [darthcloud/BlueRetro#519] [Ideas]
Hello, would there be any way to handle microphone input from a wireless controller? Right now I'm trying to design my own upgrade for the original Famicom, where it replaces the hard-wired controllers with wireless ones. Specifically the Nintendo Switch's recreated Famicom P2 controller and the PS5 controller have built-in microphones, and I think it would be really neat, besides being used in minor ways in some games.

**Comment by borishim** [comment]
Have you found a way to take care of the mic? I got interested in using the switch famicom controllers with blueretro these days

**Reply by bluestone19** [reply]
No, I haven't, sorry.
